src/parsec_client/operations/list_authenticators.md

```diff
@@ -0,0 +1,43 @@
+# ListAuthenticators
+
+Gets a list of available Parsec authenticators available for use. Opcode: 14 (`0x0014`)
+
+## Parameters
+
+No parameters are needed for this operation.
+
+## Results
+
+| Name             | Type                                                     | Description                       |
+|------------------|----------------------------------------------------------|-----------------------------------|
+| `authenticators` | Vector of [`AuthenticatorInfo`](#authenticatorinfo-type) | List of authenticator information |
+
+### AuthenticatorInfo type
+
+A `AuthenticatorInfo` type contains the following members:
+
+| Name          | Type             | Description                                                                         |
+|---------------|------------------|-------------------------------------------------------------------------------------|
+| `description` | String           | Short description of the authenticator                                              |
+| `version_maj` | Unsigned integer | Authenticator implementation version major                                          |
+| `version_min` | Unsigned integer | Authenticator implementation version minor                                          |
+| `version_rev` | Unsigned integer | Authenticator implementation version revision number                                |
+| `id`          | Unsigned integer | Authenticator ID to use on the wire protocol to communicate with this authenticator |
+
+## Specific response status codes
+
+No specific response status codes returned.
+
+## Description
+
+The version triplet returned by this operation (`version_maj`, `version_min` and `version_rev`) is
+the implementation version of the specific Parsec authenticator.
+
+The `authenticators` vector returned is in no specific order. The core authenticator will always
```

Comment:
I think this operation should be ordered, the first authenticator returned being considered as the primary one.

Reply:
ACK, will amend.